src/style-sheets/layout/_app.scss

```diff
@@ -2,25 +2,17 @@
   width: 100%;
   
     &-body {
-        max-width: 1440px;
-        margin: 0 auto;
-  
-        background-color: $olLTPrimary;
-        display: flex;
-        flex-direction: column;
-        padding-top: 5%;
-      }
-}
+      max-width: 1440px;
+      margin: 0 auto;
 
-@media only screen and (max-width: 1100px) {
-  .app-body {
-      padding-top: 9%;
-  }
-}
+      background-color: $olLTPrimary;
+      display: flex;
+      flex-direction: column;
+      padding-top: 100px;
 
-@media only screen and (max-width: 635px) {
-  .app-body {
-    width: 100%;
-    padding-top: 3%;
-  }
-}
+      @include medium('down') {
```

Comment:
I've seen the media query been done in the sass before. What's your reasoning for it? Just keep similar css next to the other styling? We usually keep it at the bottom but i could be interested in changing how we do it

Reply:
Hey! The idea is to have all the styles/behaviours together so it's easier to debug or add new styles. In my experience is much convenient to have it this way. I can move it to bottom though.

Reply:
As someone who is refactoring our CSS to be mobile design first i can actually see the benefit of this, leave it as it is and we'll revisit. It's a bitch to  figure out where all the css goes to so it makes sense to me

Reply:
I can take care of refactoring the media query if you want.